### PR TITLE
standalone vault test

### PR DESCRIPTION
**Pull request by anniehedgpeth**
## Background

[Asana Task](https://app.asana.com/0/1181500399442529/1201068889880715/f)

This branch makes the following changes:
* adds a test scenario for a standalone TFE installation with an external Vault which uses HCP vault
* adds the external vault config to the Replicated settings
* fixes the `disable-replicated-ui` argument in the TFE installation script
* removed defaults from `user_data` module
* rearranged some variables and added commented titles to beautify a bit

This branch is now working per [this PR](https://github.com/hashicorp/ptfe-replicated/pull/807#pullrequestreview-793407108) to `ptfe-replicated` which added the `VAULT_NAMESPACE` environment variable to the Replicated config. 

## How Has This Been Tested

[Passing `ptfe-replicated` CircleCI test ](https://app.circleci.com/pipelines/github/hashicorp/ptfe-replicated/2255/workflows/73dc47f2-8861-440c-927f-41b5d22fae60/jobs/18218)

## This PR makes me feel

it's about damn time

### CODE REVIEW

File: tests/standalone-vault/provider.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+provider "aws" {
+  region = local.region
```
Comment (aaron-lane) [inline tests/standalone-vault/provider.tf:2]: We can rely on `AWS_DEFAULT_REGION` being set for the provider once https://github.com/hashicorp/ptfe-replicated/pull/820 is merged.

**Review by aaron-lane — APPROVED** [review]
It's fantastic how this came together 🔐

File: tests/standalone-vault/provider.tf (hunk 1 above)
Comment (anniehedgpeth) [reply tests/standalone-vault/provider.tf:2]: Cool, I'll submit another PR after merging.

**Pull request merged**